src/main/java/model/JSONable.java

```diff
@@ -7,9 +7,11 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 
+import model.config.Config;
+
 public interface JSONable {
 
-	public static Logger logger = Logger.getLogger(Entity.class.getName());
+	public static Logger logger = Logger.getLogger(Config.class.getName());
```

Comment:
Config does not make sense I think, can you put a string in here, e.g., "JSONable"?